src/mdoc/model/DeviceResponse.ts

```diff
@@ -65,13 +65,30 @@ export class DeviceResponse {
   }
 
   /**
-   * Set the handover data to use for the device response.
+   * Set the session transcript data to use for the device response with the given handover data.
+   * this is a shortcut to calling `usingSessionTranscriptBytes(<cbor encoding of [null, null, handover] in a Tagged 24 structure>)`,
+   * which is what the OID4VP protocol expects.
    *
-   * @param {string[]} handover - The handover data to use for the device response.
+   * @param {string[]} handover - The handover data to use in the session transcript.
    * @returns {DeviceResponse}
    */
   public usingHandover(handover: string[]): DeviceResponse {
-    this.handover = handover;
+    this.usingSessionTranscriptBytes(cborEncode(DataItem.fromData([null, null, handover])));
```

Comment:
minor: include a comment saying that the null elements are for `deviceEngagementBytes` and `eReaderKeyBytes`

Reply:
👍🏼 see 53375ce